tools/admin-console/templates/index.html

```diff
@@ -13,6 +13,19 @@ <h2>Authorized Applications</h2>
 </ul>
 <a href="/app?apn="  class="btn btn-outline-primary">Create new Authorized Application</a>
 
+<hr/>
+<h2>Health Authority and Health Authority Keys</h2>
+<ul>
+  {{if .healthauthorities}}
+    {{range .healthauthorities}}
+      <li><a href="/healthauthority/{{.ID}}">{{.Issuer}}</a> | {{.Name}}</li>
+    {{end}}
+  {{else}}
+    <li><i>There are no health authorities configured</i><li>
+  {{end}}
+</ul>
+<a href="/healthauthority/0"  class="btn btn-outline-primary">Create new Health Authority</a>
```

Comment:
- Should this be `/0`?
- Nit: there's two spaces here between classanames

Reply:
yes, 0 indicates new